src/cljs/vanilla/login.cljs

```diff
@@ -82,6 +82,11 @@
          [:button.button {:on-click #(reset! @show-pop-up false)} "Cancel"]]]])))
 
 
+;@TODO - is this needed?
```

Comment:
I believe this was a change I added to stop the list of services and such leaking before the user logs in.

Reply:
Got it, put this in the fn description